doc/developers/contributing.rst

```diff
@@ -356,10 +356,19 @@ complies with the following rules before marking a PR as ``[MRG]``. The
    non-regression tests should fail for the code base in the master branch
    and pass for the PR code.
 
-5. **Make sure that your PR does not add PEP8 violations**. On a Unix-like
-   system, you can run `make flake8-diff`. `flake8 path_to_file`, would work
-   for any system, but please avoid reformatting parts of the file that your
-   pull request doesn't change, as it distracts from code review.
+5. **Make sure that your PR does not add PEP8 violations**. To check only the
+   code that you changed, you can run the following command::
+
+        git diff upstream/master -u -- "*.py" | flake8 --diff
+
+   which requires you to have set up the remote `upstream` branch to the main
+   scikit-learn repository::
+
+        $ git remote add upstream https://github.com/scikit-learn/scikit-learn.git
```

Comment:
I'd remove. The remote should be properly added already, it's in the guidelines above. IMO it's fair to assume contributors have followed them, and we need to keep the section reasonably short

Reply:
or we could point them to the section which they should have followed. In my experience people don't follow those steps.